website/docs/language/settings/backends/index.html.md

```diff
@@ -6,8 +6,11 @@ description: "A backend defines where Terraform stores its state. Learn about ho
 
 # Backends
 
-Each Terraform configuration can specify a backend, which defines where
-[state](/docs/language/state/index.html) snapshots are stored.
+Backends store [state](/docs/language/state/index.html) snapshots.
```

Comment:
I like the 'what they are' sentence followed by 'how they're defined' breakup you did here, though I prefer the original 'which defines' wording that was here. How about:

> Backends define where Terraform's state snapshots are stored.
>
> A given Terraform configuration can either specify a backend, integrate with Terraform Cloud,
or do neither and default to storing state locally.

define/determine/dictate/control/whatever

Reply:
Yeah, that's good!